network/index.md

```diff
@@ -120,6 +120,23 @@ as follows:
 2. The container uses the first response returned by any of the nameservers.
    Even if the first response is `NXDOMAIN`, or similar.
 
+### IPv6 name resolution
+
+The embedded DNS server handles both IPv4 and IPv6 name resolution. However,
+there is a caveat in name resolution for IPv6.
+
+Any IPv6 addresses specified in the `/etc/resolv.conf` file on the host system
+get copied over to the `/etc/resolv.conf` file in containers that you run.
```

Comment:
Hm.. do we do this as well if we use the embedded DNS? Or does this only apply to the default `bridge` network? @akerouanton ?

Reply:
@thaJeztah Nope, it's not only for the `bridge` network unfortunately.